anax-in-k8s/Dockerfile.ubi.amd64

```diff
@@ -11,6 +11,7 @@ LABEL description="The agent in a container that is used solely for the purpose
 ARG REQUIRED_RPMS="openssl ca-certificates shadow-utils jq iptables vim-minimal psmisc procps-ng tar"
 RUN  microdnf update  -y --nodocs --setopt=install_weak_deps=0 --disableplugin=subscription-manager \
   && microdnf install -y --nodocs --setopt=install_weak_deps=0 --disableplugin=subscription-manager ${REQUIRED_RPMS} \
+  && microdnf upgrade -y --nodocs --setopt=install_weak_deps=0 --disableplugin=subscription-manager krb5-libs \
```

Comment:
I guess you should call upgrade before install.This ensures that your system’s dependencies are up-to-date, which can prevent issues with installing new packages.

Reply:
I think that it does not matter, `upgrade` will bump everything it finds that can be upgraded